diff --git a/activity/activity.info b/activity/activity.info
index bc84f0d..59709f1 100644
--- a/activity/activity.info
+++ b/activity/activity.info
@@ -2,7 +2,16 @@
 name = Tic-Tac-Toe
 activity_version = 2
 bundle_id = org.sugarlabs.TicTacToe
+release_date = 2020-12-15
 exec = sugar-activity3 activity.TicTacToe
 icon = activity-tictactoe
 license = GPLv3+
-repository = https://github.com/devmrfitz/tictactoe
+metadata_license = CC0-1.0
+repository = https://github.com/sugarlabs/tictactoe.git
+summary = A Simple Tic-Tac-Toe Game
+description = A Simple Tic-Tac-Toe Game
+url = https://github.com/sugarlabs/tictactoe.git
+tags = Game
+update_contact = tch@sugarlabs.org
+developer_name = Sugar Labs Community
+screenshots = https://raw.githubusercontent.com/sugarlabs/tictactoe/main/help.jpg
